general-guides/community-wiki/ytm-troubleshooting.md

```diff
@@ -2,33 +2,28 @@
 
 
 
-# **Issues Within YT Music ReVanced Extended**
+### **1. No internet connection / Home page not loading / Account sign-in issues**
 
-> See below for patching and installation troubleshooting.
+Navigate to your device settings > Accounts > Manage accounts > Google (blue icon) > Remove account. You can sign in again if you want.
 
+> If that doesn't work you can uninstall GmsCore and then reinstall [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
 
 
-### **1. The download feature isn't working**
 
-Refer to step 4 of the [YTM guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-guide.md#downloader-setup) to understand and set up the download function in YTM Extended.
 
 
+### **2. The download feature isn't working**
 
+Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YTM Extended.
 
-### **2. I'm having issues signing in to my Google account**
 
-Navigate to your device settings > Accounts > Manage accounts > Vanced MicroG > Remove account. You can sign in again if you want.
 
-> If that doesn't work you can try to uninstall Vanced MicroG and reinstall [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases/latest) or [inotia00's Vanced MicroG fork](https://github.com/inotia00/VancedMicroG/releases/latest).
 
 
+### **3. How do I get YTM Extended on Android Auto?**

```

Comment:
```suggestion
### **3. How do I get YT Music RVX on Android Auto?**

```

Reply:
Is the app name really YT Music RVX? Bit convoluted huh?